collect_app/src/test/java/org/odk/collect/android/activities/FormMapViewModelTest.java

```diff
@@ -133,6 +133,30 @@
         assertThat(instances.get(6).getClickAction(), is(FormMapViewModel.ClickAction.OPEN_EDIT));
     }
 
+    @Test public void clearingInstanceGeometry_isReflectedInInstanceCountsAndList() {
```

Comment:
I'm wondering if this test could be made a little simpler if it was just `getMappableFormInstances_alwaysReturnsUpToDateInstances()`. If the `TestInstancesRepository` had a method that let you switch out the list of instances you could change instances in the repo that way and then check `getMappableFormInstances`

Reply:
Ah yes, I am far from mastering the art of writing as narrow of a test as possible. I tend to go with what matches the user experience where possible. I unfortunately won’t get a chance to come back to this in the next couple of days but if someone wants to push that change to my branch they are welcome to.